src/ToDo/Presentation/TaskViewModel.cs

```diff
@@ -43,4 +47,24 @@ private async ValueTask DoDelete(ToDoTask task, CancellationToken ct)
 			await _navigator.NavigateBackAsync(this, cancellation: ct);
 		}
 	}
+
+	public ICommand AddDueDate => Command.Create(c => c.Given(Entity).Then(DoAddDueDate));
+	private async ValueTask DoAddDueDate(ToDoTask task, CancellationToken ct)
+	{
+		var response = await _navigator.NavigateViewModelForResultAsync<ExpirationDateViewModel.BindableExpirationDateViewModel, DateTimeData>(this, qualifier:Qualifiers.Dialog, data: task, cancellation: ct);
+		if (response is null)
+		{
+			return;
+		}
+
+		var result = await response.Result;
+
+		var date = result.SomeOrDefault()?.DateTime;
+		if (date is not null)
+		{
+			var updated = task with { DueDateTime = (task.DueDateTime ?? new()) with { DateTime = (DateTime)date, TimeZone="UTC" } };
```

Comment:
Why are setting the `TimeZone` to "UTC" ? I would assume it should be local ... and actually if possible I would not specify it.